include/view/view.hpp

```diff
@@ -4,14 +4,19 @@
 
 #include <vector>
 #include <view/frame.hpp>
+#include <view/opengl_interfacing/camera.hpp>
 
 class View
 {
 protected:
 	std::vector<Frame> frames;
+
+    Camera* camera;
```

Comment:
A view doesn't need to have a camera, we have discussed it. Please move this variable to the derived class, and also please use smart pointers instead of plain ones.